stacker/plan.py

```diff
@@ -242,6 +246,9 @@ def execute(self):
                     if attempts == 1:
                         self._check_point()
                     self._wait_func(self.sleep_time)
+        except CancelExecution:
+            logger.info("Cancelling execution")
```

Comment:
i'll keep this in mind, i might make some changes to the output, i'll take a look at this then